source/NVDAObjects/window/excel.py

```diff
@@ -54,28 +54,48 @@
 xlNone=-4142
 xlSimple=-4154
 xlExtended=3
-xlCenter=-4108
-xlJustify=-4130
-xlLeft=-4131
-xlRight=-4152
-xlDistributed=-4117
-xlBottom=-4107
-xlTop=-4160
+
+# XlHAlign enumeration  from https://docs.microsoft.com/en-us/office/vba/api/excel.xlhalign
+xlHAlignCenter = -4108
+xlHAlignCenterAcrossSelection = 7
+xlHAlignDistributed = -4117
+xlHAlignFil = 5
+xlHAlignGeneral = 1
+xlHAlignJustify = -4130
+xlHAlignLeft = -4131
+xlHAlignRight = -4152
+
+# XlVAlign enumeration from https://docs.microsoft.com/en-us/office/vba/api/excel.xlvalign
+xlVAlignBottom = -4107
+xlVAlignCenter = -4108
+xlVAlignDistributed = -4117
+xlVAlignJustify = -4130
+xlVAlignTop = -4160
+

```

Comment:
can these become 2 separate `IntEnum`s? `horizontalAlignmentLabels` and `verticalAlignmentLabels` can be their displayStringLabels